truss/util/path.py

```diff
@@ -1,33 +1,57 @@
+import fnmatch
 import os
 import random
 import string
 import tempfile
 from contextlib import contextmanager
-from distutils.dir_util import copy_tree, remove_tree
 from distutils.file_util import copy_file
 from pathlib import Path
-from typing import List, Optional, Tuple, Union
+from shutil import copytree
+from typing import List, Optional, Tuple
 
 from truss.patch.hash import str_hash_str
 
+TRUSS_IGNORE_PATH = Path(__file__).parent / ".truss_ignore"
```

Comment:
I got really confused for a bit, but then I understood. We don't support any user supplied `.truss_ignore`, this is referring to the fixed .truss_ignore that we use. It would be good to rename this for to say `FIXED_TRUSS_IGNORE_PATH` and add a comment explaining that this is a fixed file in the truss library, for now.